addons/web/static/src/core/l10n/utils/format_list.js

```diff
@@ -71,5 +71,5 @@ const LIST_STYLES = {
 export function formatList(list, { localeCode = "", style = "standard" } = {}) {
     const locale = localeCode || user.lang || "en-US";
     const formatter = new Intl.ListFormat(locale, LIST_STYLES[style]);
-    return formatter.format(list);
+    return formatter.format(list.map(String));
```

Comment:
`map` is only available on Array, if all iterable objects are supported, this cannot be used.
If you need filtering or transforming you probably need to use for-loop and re-build a new list.

Reply:
Thanks. I've updated the code to turn it into an Array of strings.